parser/src/aspects/call.rs

```diff
@@ -26,6 +29,15 @@ pub trait CallAspect<'a> {
     /// Parses a programmatic call.
     fn programmatic_call(&mut self) -> ParseResult<Expr<'a>>;
 
+    /// Parses a programmatic call with a known command name expression.
+    fn programmatic_call_on(&mut self, expr: Expr<'a>) -> ParseResult<Expr<'a>>;
```

Comment:
This makes no sense to have a _function call_ on an expression (which thus is a method call) if you already defined structures for them